PermissionsEx/permissions.yml

```diff
@@ -7277,6 +7277,26 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.lightning
+  electro:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&bE&1l&be&1c&bt&1r&bo'
```

Comment:
There are two electros, Jamie Foxx and Francine Frye, which one is this? You will have to update